elektronn3/data/transforms/transforms.py

```diff
@@ -436,6 +496,131 @@ def __call__(
         return inp_cropped, target_cropped
 
 
+class ElasticTransform:
+    """
+    Based on https://gist.github.com/fmder/e28813c1e8721830ff9c
+
+
+    Elastic deformation of images as described in [Simard2003]_.
+    .. [Simard2003] Simard, Steinkraus and Platt, "Best Practices for
+       Convolutional Neural Networks applied to Visual Document Analysis", in
+       Proc. of the International Conference on Document Analysis and
+       Recognition, 2003.
+
+        Args:
+            sigma: Sigma parameter of the gaussian distribution to draw from
+            alpha: Strength of the elastic transform
+            rng: Optional random state for deterministic execution
+            channels: If ``channels`` is ``None``, the change is applied to
+                all channels of the input tensor.
+                If ``channels`` is a ``Sequence[int]``, change is only applied
+                to the specified channels.
+            prob: probability (between 0 and 1) with which to perform this
+                augmentation. The input is returned unmodified with a probability
+                of ``1 - prob``
+            target_discrete_ix: list
+                List of target channels that contain discrete values.
+                By default (``None``), every channel is is seen as discrete (this is
+                generally the case for classification tasks).
+                This information is used to decide what kind of interpolation should
+                be used for reading target data:
+
+                    - discrete targets are obtained by nearest-neighbor interpolation
+                    - non-discrete (continuous) targets are linearly interpolated.
+
+        The input image should be of dimensions (C, H, W) or (C, D, H, W).
+        C must be included.
+
+    """
+
+    def __init__(
+            self,
+            sigma: float = 4,
+            alpha: float = 10,
+            channels: Optional[Sequence[int]] = None,
+            prob: float = 0.25,
+            rng: Optional[np.random.RandomState] = None,
+            target_discrete_ix: Optional[list]= None,
+
+    ):
+        self.sigma = sigma
+        self.alpha = alpha
+        self.channels = channels
+        self.prob = prob
+        self.rng = np.random.RandomState() if rng is None else rng
+        self.target_discrete_ix = target_discrete_ix
+
+    def __call__(
+            self,
+            inp: np.ndarray,
+            target: Optional[np.ndarray] = None  # returned without modifications
```

Comment:
```suggestion
            target: Optional[np.ndarray] = None
```
It's not returned without modification if it's not `None`